tests/AppBundle/Helper/AutomatedEditsTest.php

```diff
@@ -32,7 +32,6 @@ public function setUp(): void
         $client = static::createClient();
         $container = $client->getContainer();
         $this->aeh = new AutomatedEditsHelper($container);
-        $this->cache = $container->get('cache.app');
```

Comment:
Same as some of my other comments - how is this related to this patch / can it be done separately?